Sources/WebSocketKit/WebSocket.swift

```diff
@@ -24,7 +30,11 @@ public final class WebSocket {
         self.channel.closeFuture
     }
 
+    public var jsonDecoder = JSONDecoder()
+    public var jsonEncoder = JSONEncoder()
+
```

Comment:
Lets keep these private for now. If we want them to be configurable we should instead allow them to be passed in to the methods where they are used and default to new ones

Reply:
Ok changed them to private